datafusion/physical-plan/src/aggregates/group_values/row.rs

```diff
@@ -199,6 +203,20 @@ impl GroupValues for GroupValuesRows {
             }
         };
 
+        // TODO: Materialize dictionaries in group keys
```

Comment:
Do you know if this is described in a ticket yet (I think the answer is no, but I am not 100% sure)

Reply:
No a ticket would be a good addition

Reply:
Created #7647